similarity-genotype-view/src/main/java/org/phenotips/data/similarity/genotype/ExomiserExome.java

```diff
@@ -96,10 +96,6 @@
             }
 
             geneMutations.add(variant);
-            // Represent homozygous variants as two separate mutations
```

Comment:
any reason to change this logic? do you know why we had it like this in the first place?

Reply:
Clearly labeling the variant as "homozygous" in the UI is easier for the user to process than duplicating the variant and letting the user notice there are two identical entries.

The duplication of homozygous variants was introduced by @buske [6 years ago](https://github.com/phenotips/patient-network/blame/f4cef44d8389e62365651fb2823748c94c8a364e/similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/ExomizerGenotype.java#L81).

Reply:
Oh goodness, yes, it was the easiest approach at the time... sorry.